package.json

```diff
@@ -13,9 +13,8 @@
     "npm": "1.2.x"
   },
   "scripts": {
-    "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
-    "postinstall": "node node_modules/grunt-cli/bin/grunt install"
+    "start": "gulp",
```

Comment:
consider taking another look at this. What happens when gulp isn't installed as a global dependency in your production environment. Do you think this command will run?

Reply:
I dint think over that instance, so i will refactor to include it as dependencies so as cater for production.

Reply:
This won't still work. Except gulp installed globally. To be on a safer side, you can read more about running npm local packages [here](https://firstdoit.com/no-need-for-globals-using-npm-dependencies-in-npm-scripts-3dfb478908)